### ISSUE TITLE
Process finished with exit code -1073741819 (0xC0000005)

### ISSUE THREAD

**Issue by ymzx** [PaddlePaddle/PaddleOCR#2917]
windos10机器,
当img_shape = (1, 3, 16, 114)时  运行到predict_det.py 脚本程序自动退出
通过debug可以定位到
self.input_tensor.copy_from_cpu(img)和self.predictor.run()之间，
即程序运行到self.predictor.run()自动退出,报错如下
Process finished with exit code -1073741819 (0xC0000005)
请问可能是什么原因？
paddle==2.0.2,CPU环境

**Comment by MissPenguin** [comment]
可以先试试以下2个方式：
1. 更新paddle到2.1.0
2. tools/infer/utility.py的create_predictor里你加一行这个试试：config.switch_ir_optim(False)

**Comment by hhhhhhogan** [comment]
@MissPenguin @ymzx  您好，我这里也遇到了这个问题，目前paddle是2.1.1，tools/infer/utility.py的create_predictor 也加了config.switch_ir_optim(False) ，我加在了149行 config = inference.Config(model_file_path, params_file_path) 的下边一行，
之前运行代码显示：
Process finished with exit code -1073741819 (0xC0000005)
现在就没有了，不过感觉没有执行成功
执行代码为：
import os
import cv2
from paddleocr import PPStructure,draw_structure_result,save_structure_res

table_engine = PPStructure(show_log=True)

save_folder = 'E:/dataset/pdf_table/table_output'
img_path = 'E:/dataset/pdf_table/pdf_output/1.png'
# img_path = 'doc/table/1.png'
img = cv2.imread(img_path)
result = table_engine(img)
save_structure_res(result, save_folder, os.path.basename(img_path).split('.')[0])

for line in result:
    line.pop('img')
    print(line)

from PIL import Image

font_path = '../doc/fonts/simfang.ttf' # PaddleOCR下提供字体包
image = Image.open(img_path).convert('RGB')
im_show = draw_structure_result(image, result,font_path=font_path)
im_show = Image.fromarray(im_show)
im_show.save('result.jpg')

运行的是表格识别部分，环境为win10，目前使用的是cpu环境，


提示信息为：

D:\anaconda\envs\pp\lib\site-packages\win32\lib\pywintypes.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp, sys, os
Namespace(benchmark=False, cls_batch_num=6, cls_image_shape='3, 48, 192', cls_model_dir=None, cls_thresh=0.9, cpu_threads=10, det=True, det_algorithm='DB', det_db_box_thresh=0.6, det_db_score_mode='fast', det_db_thresh=0.3, det_db_unclip_ratio=1.5, det_east_cover_thresh=0.1, det_east_nms_thresh=0.2, det_east_score_thresh=0.8, det_limit_side_len=960, det_limit_type='max', det_model_dir='C:\\Users\\Dell/.paddleocr/2.2.0.1\\ocr\\det\\ch\\ch_ppocr_mobile_v2.0_det_infer', det_sast_nms_thresh=0.2, det_sast_polygon=False, det_sast_score_thresh=0.5, drop_score=0.5, e2e_algorithm='PGNet', e2e_char_dict_path='./ppocr/utils/ic15_dict.txt', e2e_limit_side_len=768, e2e_limit_type='max', e2e_model_dir=None, e2e_pgnet_mode='fast', e2e_pgnet_polygon=True, e2e_pgnet_score_thresh=0.5, e2e_pgnet_valid_set='totaltext', enable_mkldnn=False, gpu_mem=500, help='==SUPPRESS==', image_dir=None, ir_optim=True, label_list=['0', '180'], lang='ch', layout_path_model='lp://PubLayNet/ppyolov2_r50vd_dcn_365e_publaynet/config', max_batch_size=10, max_text_length=25, min_subgraph_size=10, output='./output/table', precision='fp32', process_id=0, rec=True, rec_algorithm='CRNN', rec_batch_num=6, rec_char_dict_path='E:\\temp_project\\PaddleOCR-release-2.2\\ppocr\\utils\\ppocr_keys_v1.txt', rec_char_type='ch', rec_image_shape='3, 32, 320', rec_model_dir='C:\\Users\\Dell/.paddleocr/2.2.0.1\\ocr\\rec\\ch\\ch_ppocr_mobile_v2.0_rec_infer', save_log_path='./log_output/', show_log=True, table_char_dict_path='E:\\temp_project\\PaddleOCR-release-2.2\\ppocr\\utils\\dict\\table_structure_dict.txt', table_char_type='en', table_max_len=488, table_model_dir='C:\\Users\\Dell/.paddleocr/2.2.0.1\\ocr\\table\\en_ppocr_mobile_v2.0_table_structure_infer', total_process_num=1, type='ocr', use_angle_cls=False, use_dilation=False, use_gpu=True, use_mp=False, use_pdserving=False, use_space_char=True, use_tensorrt=False, vis_font_path='./doc/fonts/simfang.ttf', warmup=True)
E0901 15:32:07.241871 27560 analysis_config.cc:81] Please compile with gpu to EnableGpu()
e[1me[35m--- Running analysis [ir_graph_build_pass]e[0m
e[1me[35m--- Running analysis [ir_graph_clean_pass]e[0m
e[1me[35m--- Running analysis [ir_analysis_pass]e[0m
e[32m--- Running IR pass [simplify_with_basic_ops_pass]e[0m
e[32m--- Running IR pass [layer_norm_fuse_pass]e[0m
e[37m---    Fused 0 subgraphs into layer_norm op.e[0m
e[32m--- Running IR pass [attention_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [seqconv_eltadd_relu_fuse_pass]e[0m
e[32m--- Running IR pass [seqpool_cvm_concat_fuse_pass]e[0m
e[32m--- Running IR pass [mul_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [fc_gru_fuse_pass]e[0m
e[32m--- Running IR pass [mul_gru_fuse_pass]e[0m
e[32m--- Running IR pass [seq_concat_fc_fuse_pass]e[0m
e[32m--- Running IR pass [squeeze2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [reshape2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [flatten2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [map_matmul_to_mul_pass]e[0m
e[32m--- Running IR pass [fc_fuse_pass]e[0m
e[32m--- Running IR pass [repeated_fc_relu_fuse_pass]e[0m
e[32m--- Running IR pass [squared_mat_sub_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0901 15:32:07.329635 27560 graph_pattern_detector.cc:91] ---  detected 33 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [is_test_pass]e[0m
e[32m--- Running IR pass [runtime_context_cache_pass]e[0m
e[1me[35m--- Running analysis [ir_params_sync_among_devices_pass]e[0m
e[1me[35m--- Running analysis [adjust_cudnn_workspace_size_pass]e[0m
e[1me[35m--- Running analysis [inference_op_replace_pass]e[0m
e[1me[35m--- Running analysis [memory_optimize_pass]e[0m
I0901 15:32:07.352574 27560 memory_optimize_pass.cc:199] Cluster name : batch_norm_4.tmp_2  size: 13107200
I0901 15:32:07.352574 27560 memory_optimize_pass.cc:199] Cluster name : tmp_2  size: 9830400
I0901 15:32:07.352574 27560 memory_optimize_pass.cc:199] Cluster name : batch_norm_44.tmp_1  size: 1920
I0901 15:32:07.352574 27560 memory_optimize_pass.cc:199] Cluster name : conv2d_60.tmp_0  size: 13107200
I0901 15:32:07.352574 27560 memory_optimize_pass.cc:199] Cluster name : x  size: 4915200
I0901 15:32:07.352574 27560 memory_optimize_pass.cc:199] Cluster name : elementwise_add_4  size: 256000
I0901 15:32:07.352574 27560 memory_optimize_pass.cc:199] Cluster name : depthwise_conv2d_0.tmp_0  size: 3276800
I0901 15:32:07.352574 27560 memory_optimize_pass.cc:199] Cluster name : nearest_interp_v2_4.tmp_0  size: 2457600
I0901 15:32:07.352574 27560 memory_optimize_pass.cc:199] Cluster name : elementwise_add_3  size: 614400
e[1me[35m--- Running analysis [ir_graph_to_program_pass]e[0m
I0901 15:32:07.392499 27560 analysis_predictor.cc:636] ======= optimize end =======
I0901 15:32:07.392499 27560 naive_executor.cc:98] ---  skip [feed], feed -> x
I0901 15:32:07.394491 27560 naive_executor.cc:98] ---  skip [batch_norm_4.tmp_2], fetch -> fetch
E0901 15:32:07.399482 27560 analysis_config.cc:81] Please compile with gpu to EnableGpu()
e[1me[35m--- Running analysis [ir_graph_build_pass]e[0m
e[1me[35m--- Running analysis [ir_graph_clean_pass]e[0m
e[1me[35m--- Running analysis [ir_analysis_pass]e[0m
e[32m--- Running IR pass [simplify_with_basic_ops_pass]e[0m
e[32m--- Running IR pass [layer_norm_fuse_pass]e[0m
e[37m---    Fused 0 subgraphs into layer_norm op.e[0m
e[32m--- Running IR pass [attention_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [seqconv_eltadd_relu_fuse_pass]e[0m
e[32m--- Running IR pass [seqpool_cvm_concat_fuse_pass]e[0m
e[32m--- Running IR pass [mul_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [fc_gru_fuse_pass]e[0m
e[32m--- Running IR pass [mul_gru_fuse_pass]e[0m
e[32m--- Running IR pass [seq_concat_fc_fuse_pass]e[0m
e[32m--- Running IR pass [squeeze2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [reshape2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [flatten2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [map_matmul_to_mul_pass]e[0m
I0901 15:32:07.448318 27560 graph_pattern_detector.cc:91] ---  detected 1 subgraphs
e[32m--- Running IR pass [fc_fuse_pass]e[0m
I0901 15:32:07.449316 27560 graph_pattern_detector.cc:91] ---  detected 1 subgraphs
e[32m--- Running IR pass [repeated_fc_relu_fuse_pass]e[0m
e[32m--- Running IR pass [squared_mat_sub_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0901 15:32:07.472283 27560 graph_pattern_detector.cc:91] ---  detected 24 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [is_test_pass]e[0m
e[32m--- Running IR pass [runtime_context_cache_pass]e[0m
e[1me[35m--- Running analysis [ir_params_sync_among_devices_pass]e[0m
e[1me[35m--- Running analysis [adjust_cudnn_workspace_size_pass]e[0m
e[1me[35m--- Running analysis [inference_op_replace_pass]e[0m
e[1me[35m--- Running analysis [memory_optimize_pass]e[0m
I0901 15:32:07.494195 27560 memory_optimize_pass.cc:199] Cluster name : save_infer_model/scale_0.tmp_1  size: 662500
I0901 15:32:07.494195 27560 memory_optimize_pass.cc:199] Cluster name : batch_norm_12.tmp_2  size: 19200
I0901 15:32:07.494195 27560 memory_optimize_pass.cc:199] Cluster name : softmax_0.tmp_0  size: 662500
I0901 15:32:07.494195 27560 memory_optimize_pass.cc:199] Cluster name : batch_norm_29.tmp_2  size: 115200
I0901 15:32:07.494195 27560 memory_optimize_pass.cc:199] Cluster name : elementwise_add_1  size: 19200
I0901 15:32:07.494195 27560 memory_optimize_pass.cc:199] Cluster name : x  size: 38400
I0901 15:32:07.494195 27560 memory_optimize_pass.cc:199] Cluster name : lstm_0.tmp_2  size: 768
I0901 15:32:07.494195 27560 memory_optimize_pass.cc:199] Cluster name : lstm_0.tmp_3  size: 1
I0901 15:32:07.494195 27560 memory_optimize_pass.cc:199] Cluster name : lstm_0._generated_var_0  size: 1
e[1me[35m--- Running analysis [ir_graph_to_program_pass]e[0m
I0901 15:32:07.541070 27560 analysis_predictor.cc:636] ======= optimize end =======
I0901 15:32:07.541070 27560 naive_executor.cc:98] ---  skip [feed], feed -> x
I0901 15:32:07.545059 27560 naive_executor.cc:98] ---  skip [save_infer_model/scale_0.tmp_1], fetch -> fetch
E0901 15:32:07.546061 27560 analysis_config.cc:81] Please compile with gpu to EnableGpu()
e[1me[35m--- Running analysis [ir_graph_build_pass]e[0m
e[1me[35m--- Running analysis [ir_graph_clean_pass]e[0m
e[1me[35m--- Running analysis [ir_analysis_pass]e[0m
e[32m--- Running IR pass [simplify_with_basic_ops_pass]e[0m
e[32m--- Running IR pass [layer_norm_fuse_pass]e[0m
e[37m---    Fused 0 subgraphs into layer_norm op.e[0m
e[32m--- Running IR pass [attention_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [seqconv_eltadd_relu_fuse_pass]e[0m
e[32m--- Running IR pass [seqpool_cvm_concat_fuse_pass]e[0m
e[32m--- Running IR pass [mul_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [fc_gru_fuse_pass]e[0m
e[32m--- Running IR pass [mul_gru_fuse_pass]e[0m
e[32m--- Running IR pass [seq_concat_fc_fuse_pass]e[0m
e[32m--- Running IR pass [squeeze2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [reshape2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [flatten2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [map_matmul_to_mul_pass]e[0m
I0901 15:32:07.633852 27560 graph_pattern_detector.cc:91] ---  detected 3 subgraphs
e[32m--- Running IR pass [repeated_fc_relu_fuse_pass]e[0m
e[32m--- Running IR pass [squared_mat_sub_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0901 15:32:07.669726 27560 graph_pattern_detector.cc:91] ---  detected 32 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [is_test_pass]e[0m
e[32m--- Running IR pass [runtime_context_cache_pass]e[0m
e[1me[35m--- Running analysis [ir_params_sync_among_devices_pass]e[0m
e[1me[35m--- Running analysis [adjust_cudnn_workspace_size_pass]e[0m
e[1me[35m--- Running analysis [inference_op_replace_pass]e[0m
e[1me[35m--- Running analysis [memory_optimize_pass]e[0m
I0901 15:32:07.701640 27560 memory_optimize_pass.cc:199] Cluster name : relu_2.tmp_0  size: 15241216
I0901 15:32:07.701640 27560 memory_optimize_pass.cc:199] Cluster name : batch_norm_4.tmp_3  size: 15241216
I0901 15:32:07.701640 27560 memory_optimize_pass.cc:199] Cluster name : batch_norm_6.tmp_3  size: 1428864
I0901 15:32:07.701640 27560 memory_optimize_pass.cc:199] Cluster name : depthwise_conv2d_2.tmp_0  size: 4286592
I0901 15:32:07.701640 27560 memory_optimize_pass.cc:199] Cluster name : x  size: 2857728
I0901 15:32:07.701640 27560 memory_optimize_pass.cc:199] Cluster name : batch_norm_12.tmp_3  size: 595360
e[1me[35m--- Running analysis [ir_graph_to_program_pass]e[0m
I0901 15:32:07.781427 27560 analysis_predictor.cc:636] ======= optimize end =======
I0901 15:32:07.782425 27560 naive_executor.cc:98] ---  skip [feed], feed -> x
I0901 15:32:07.786414 27560 naive_executor.cc:98] ---  skip [batch_norm_4.tmp_3], fetch -> fetch
I0901 15:32:07.786414 27560 naive_executor.cc:98] ---  skip [relu_2.tmp_0], fetch -> fetch
E0901 15:32:07.786414 27560 analysis_config.cc:81] Please compile with gpu to EnableGpu()
e[37m---    Fused 0 subgraphs into layer_norm op.e[0m

Process finished with exit code 0
